cdc/owner.go

```diff
@@ -426,17 +428,31 @@ func (o *Owner) newChangeFeed(
 		}
 
 	}
-	errCh := make(chan error, 1)
 
-	primarySink, err := sink.NewSink(ctx, id, info.SinkURI, filter, info.Config, info.Opts, errCh)
+	cdcCtx = cdcContext.WithChangefeedVars(cdcCtx, &cdcContext.ChangefeedVars{
+		ID:   id,
+		Info: info,
+	})
+	errCh := make(chan error, 1)
+	cdcCtx = cdcContext.WithErrorHandler(cdcCtx, func(err error) error {
+		errCh <- errors.Trace(err)
+		return nil
+	})
```

Comment:
This `errCh` is consumed only once, could it happen multiple errors need to be thrown and error sending blocks here

Reply:
thanks

Reply:
done